packages/cli/lib/services/dryrun.service.ts

```diff
@@ -541,7 +569,12 @@ export class DryRunService {
                                 return unzipper;
                             case 'soap':
                             case 'botbuilder':
-                                throw new Error(`Module '${moduleName}' not available in dry run. Please test the integration using the Nango dashboard`);
+                                if (!preloaded[moduleName]) {
+                                    throw new Error(
+                                        `Module '${moduleName}' needs to be installed on your side in a package.json. You can also test using the Nango dashboard`
```

Comment:
> You can also test using the Nango dashboard
Should we be a bit more explicit and say "Deploy to Nango dev environment to test" or something similar